test/acceptance/team_test.rb

```diff
@@ -37,4 +37,32 @@ def test_managing_a_team_as_sole_manager
       end
     end
   end
+
+  def test_team_pagination
+
+    100.times do |index|
```

Comment:
What is the smallest number of users that you need that will trigger the behavior that you want to test?